pkg/networkservice/common/connect/server_test.go

```diff
@@ -62,6 +63,17 @@ func startServer(ctx context.Context, listenOn *url.URL, server networkservice.N
 	}
 }
 
+func checkServer(target *url.URL) error {
```

Comment:
Probably better to register health service on the tested server and test that server started to serve to verify the sever.